app/services/fact_importer.rb

```diff
@@ -20,7 +20,8 @@ def self.fact_features
 
   def self.authorized_smart_proxy_features
     # When writing your own Fact importer, provide feature(s) of authorized Smart Proxies
-    raise NotImplementedError
+    Rails.logger.debug("Importer #{self} does not implement authorized_smart_proxy_features.")
```

Comment:
Using ```Rails.logger``` is necessary as this will run from a plugin context where logger is undefined. See the error I got when testing this with ForemanSalt.

```
 undefined local variable or method `logger' for ForemanSalt::FactImporter:Class
```